deploy/charts/venafi-kubernetes-agent/values.yaml

```diff
@@ -45,47 +45,53 @@ metrics:
     #     sourceLabels:
     #     - __meta_kubernetes_pod_node_name
     #     targetLabel: instance
-    #
     endpointAdditionalProperties: {}
 
-# -- default replicas, do not scale up
+# default replicas, do not scale up
 replicaCount: 1
 
 image:
-  # -- Default to Open Source image repository
+  # The container image for the Venafi Enhanced Issuer manager.
   repository: registry.venafi.cloud/venafi-agent/venafi-agent
-  # -- Defaults to only pull if not already present
+
+  # Kubernetes imagePullPolicy on Deployment.
   pullPolicy: IfNotPresent
-  # -- Overrides the image tag whose default is the chart appVersion
-  # tag: "v0.0.0"
 
-# -- Specify image pull credentials if using a private registry
-# example: - name: my-pull-secret
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.0.0"
```

Comment:
@maelvls This default value makes it into the README file which may confuse users:

```console
$ helm inspect readme _bin/scratch/image/venafi-kubernetes-agent-1.1.0-dirty.tgz | fgrep -A 5 image.tag
#### **image.tag** ~ `string`
> Default value:
> ```yaml
> v0.0.0
> ```
```

`make helm-chart` uses `yq` to modify this value after running `helm package`.

Reply:
The `image.tag` value is not on the documentation website:
 * https://docs.venafi.cloud/vaas/k8s-components/c-vka-helmvalues/#imagerepository

Reply:
Well spotted, that’s a problem. 

- I’ll let Michael know about the outdated Helm reference page,
- I propose that we remove the default tag and only show an example; I can open a PR with this.

Reply:
`I propose that we remove the default tag and only show an example; I can open a PR with this.`
👍 
Sounds like a good idea, I think that will work.